frontend/src/bundles/common/components/upload-video/components/video-player/components/control/control.tsx

```diff
@@ -0,0 +1,262 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { type PlayerRef } from '@remotion/player';
+
+import {
+    Box,
+    Flex,
+    HStack,
+    Icon,
+    IconButton,
+    Slider,
+    SliderFilledTrack,
+    SliderThumb,
+    SliderTrack,
+    Text,
+} from '~/bundles/common/components/components.js';
+import {
+    type PlayerOptions,
+    type VideoDuration,
+} from '~/bundles/common/components/upload-video/components/video-player/libs/types/types.js';
+import {
+    useCallback,
+    useEffect,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+import { IconName } from '~/bundles/common/icons/icon-name.js';
+
+import { VideoEvent } from './libs/enums/enums.js';
+import { getTime } from './libs/helpers/helpers.js';
+import { useAnimationFrame } from './libs/hooks/hooks.js';
+
+type Properties = {
+    videoPlayerReference: React.RefObject<PlayerRef>;
+    duration: VideoDuration;
+};
+
+const Control: React.FC<Properties> = ({ videoPlayerReference, duration }) => {
+    const [videoState, setVideoState] = useState<PlayerOptions>({
+        isPlaying: false,
+        isMuted: false,
+        wasPlaying: true,
+        volume: 0.5,
+        played: 0,
+    });
+
+    const time = getTime({
+        played: videoState.played,
+        durationInSeconds: duration.inSeconds,
+    });
+
+    const handleVideoEnd = useCallback(() => {
+        setVideoState({
+            ...videoState,
+            isPlaying: false,
+            played: 0,
+            isMuted: videoPlayerReference.current?.isMuted() as boolean,
+            wasPlaying: true,
+        });
+    }, [videoState, videoPlayerReference]);
+
+    const updateSlider = (): void => {
+        if (videoPlayerReference.current?.isPlaying()) {
+            const currentTime =
+                videoPlayerReference.current.getCurrentFrame() /
+                duration.inFrames;
+
+            setVideoState((previousState) => {
+                return {
+                    ...previousState,
+                    played: currentTime * 100,
+                };
+            });
+        }
+    };
+
+    useAnimationFrame(updateSlider, videoState.isPlaying);
+
+    useEffect(() => {
+        const player = videoPlayerReference.current;
+        player?.addEventListener(VideoEvent.ENDED, handleVideoEnd);
+
+        return () => {
+            if (player) {
+                player.removeEventListener(VideoEvent.ENDED, handleVideoEnd);
+            }
```

Comment:
```suggestion
            player?.addEventListener(VideoEvent.ENDED, handleVideoEnd);
```
Just to be consistent